MiaWrapper.py

```diff
@@ -24,8 +24,18 @@ def miaWrapper(computationDataJson, paramsFile, outputPath):
     roiName = volumesOfInterest[0]['rois'][0]
    
     PyradiomicsWrapper.performRadiomicsComputation(ctDirectory, rtStructDirectory, roiName, outputPath, paramsFile)
-    
-    
+
+def checkIfCopyIsNeeded(images, modalities):
+    modality = modalities[0]
+    modalityImages = images[modality]
+
+    currentImageDirectory = os.path.dirname(modalityImages[0]['location'])
+    newImageDirectory = os.path.join(currentImageDirectory, modality)
+    if os.path.basename(currentImageDirectory) in modalities or os.path.isdir(newImageDirectory):
```

Comment:
This if-else looks to be equivalent to:
return os.path.basename(currentImageDirectory) in modalities or os.path.isdir(newImageDirectory)